src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -314,6 +314,72 @@ public virtual CreatedResult Created(Uri uri, object content)
             return new CreatedAtRouteResult(routeName, routeValues, content);
         }
 
+        /// <summary>
+        /// Creates a <see cref="AcceptedObjectResult"/> (202 Accepted) with the specified values.
+        /// </summary>
+        /// <param name="location">
+        /// The location at which the status of requested content can be monitored. Must be a relative or absolute URL.
+        /// </param>
+        /// <param name="content">The content value to format in the entity body.</param>
+        /// <returns>A <see cref="AcceptedObjectResult"/> with the specified values.</returns>
+        [NonAction]
+        public virtual AcceptedObjectResult Accepted(string location, object content)
```

Comment:
@rynowak @pranavkm Can someone expand on this? Is this class for legacy webapi and I do not need to add anything related to Accepted here since it doesn't exist in old stuff?